compiler/optimizer/VPHandlers.cpp

```diff
@@ -11039,17 +11039,12 @@ static TR::Node *constrainCmpeqne(OMR::ValuePropagation *vp, TR::Node *node, boo
    TR::VPConstraint *constraint = NULL;
    if (result >= 0)
       {
-      if ((lhsGlobal || vp->lastTimeThrough()) &&
-          performTransformation(vp->comp(), "%sChanging node [%p] %s into constant %d\n", OPT_DETAILS, node, node->getOpCode().getName(), result))
+      constraint = TR::VPIntConst::create(vp, result/*, isUnsigned*/);
+      if (lhsGlobal || vp->lastTimeThrough())
```

Comment:
Not necessarily a problem, but I don't think this `if` is needed anymore, since `replaceByConstant()` already checks the same condition before actually folding. At this point it should be fine to unconditionally `replaceByConstant()` and then return